modules/rosa-upgrading-cli-tutorial.adoc

```diff
@@ -0,0 +1,51 @@
+:_module-type: PROCEDURE
+//Specify the system-module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-upgrading.adoc
+
+[id="rosa-upgrading-cli_{context}"]
+= Upgrading with the rosa CLI
+
+[role="_abstract"]
+There are three methods that can be used to upgrade {product-title} clusters. This procedure describes upgrading using the `rosa` CLI.
+
+This method schedules the cluster for an immediate upgrade, if a more recent version is available.
+
+.Procedure
+. To verify that an upgrade is available, enter the following command:
++
+[source,terminal]
+----
+$ rosa list versions
+----
++
+A list displays the current version of your cluster, indicated by `true`, and a list of available versions. The following example shows that the current version is `v4.6.8`, and the most recent version is `v4.6.12`.
++
+.Example output
+[source,terminal]
+----
+ID                   DEFAULT
+openshift-v4.6.8     true
+openshift-v4.6.1     false
+openshift-v4.6.12    false
+openshift-v4.6.3     false
+openshift-v4.6.4     false
+openshift-v4.6.6     false
+----
+
+. To upgrade a cluster, enter the following command:
++
+[source,terminal]
+----
+$ rosa upgrade cluster --cluster=<cluster_name> --version=<version_number>
+----
++
+.Example
+[source,terminal]
+----
+$ rosa upgrade cluster --cluster=mycluster --version=4.6.12
+----
++
+The cluster is scheduled for an immediate upgrade. This action takes some time to complete, approximately 30 - 40 minutes. The terminal displays a confirmation message when the upgrade is complete.
```

Comment:
without splitting hairs, I'd say it is safer to say something like "may take an hour or longer depending on your workload configuration (for example, PodDisruptionBudgets)."

Reply:
Done.